ckan-dev/setup/start_ckan_development.sh

```diff
@@ -37,6 +49,20 @@ python prerun.py
 echo "Loading the following plugins: $CKAN__PLUGINS"
 paster --plugin=ckan config-tool $CKAN_INI "ckan.plugins = $CKAN__PLUGINS"
 
+# Update test-core.ini DB, SOLR & Redis settings
+echo "Loading test settings into test-core.ini"
+paster --plugin=ckan config-tool \
```

Comment:
`config-tool` accepts passing multiple options to change at the same time, which will make things more readable:

```
paster --plugin=ckan config-tool $SRC_DIR/ckan/test-core.ini \
    "sqlalchemy.url = $TEST_CKAN_SQLALCHEMY_URL" \
    "ckan.datstore.write_url = $TEST_CKAN_DATASTORE_WRITE_URL" \
    "ckan.datstore.read_url = $TEST_CKAN_DATASTORE_READ_URL" \
    "solr_url = http://solr:8983/solr/ckan" \
    "ckan.redis_url = redis://redis:6379/1"
```